okio/jvm/src/main/java/okio/ByteString.kt

```diff
@@ -227,7 +227,11 @@ internal constructor(
   operator fun get(index: Int): Byte = getByte(index)
 
   /** Returns the number of bytes in this ByteString. */
-  open fun size() = data.size
+  val size
+    @JvmName("size") get() = getSize()
+
+  // Hack to work around Kotlin's limitation for using JvmName on open/override vals/funs
+  protected open fun getSize() = data.size
```

Comment:
On a second thought, `internal` might be a better option? This way it shouldn't appear in KDoc